src/js/view/controls/rightclick.js

```diff
@@ -156,6 +165,9 @@ export default class RightClick {
             this.el.addEventListener('mouseout', this.outHandler);
         }
         this.el.querySelector('.jw-info-overlay-item').addEventListener('click', this.infoOverlayHandler);
+        if (this.shareOnRightClick) {
```

Comment:
We don't need `this.shareOnRightClick`.

You could do this, but you should **use `UI` for tap support**.
```js
const shareItemElement = this.el.querySelector('.jw-share-item');
if (shareItemElement) {
   shareItemElement.addEventListener('click', this.shareHandler);
}
```